docs/pages/web/sidenavigation.js

```diff
@@ -454,6 +459,53 @@ When building SideNavigation, we might want to render different combinations of
           />
         </MainSection.Subsection>
 
+        <MainSection.Subsection title="Collapsible" description="">
+          <MainSection.Card
+            cardSize="lg"
+            title="Items with icons"
+            sandpackExample={
+              <SandpackExample
+                code={collapsibleExample}
+                name="Collapsible with icons example"
+                previewHeight={500}
+              />
+            }
+          />
+          <MainSection.Card
+            cardSize="lg"
+            title="Items without icons"
+            sandpackExample={
+              <SandpackExample
+                code={collapsibleWithoutIconsExample}
+                name="Collapsible without icons example"
+                previewHeight={500}
+              />
+            }
+          />
+          <MainSection.Card
+            cardSize="lg"
+            title="Mixed items with and without icons"
+            sandpackExample={
+              <SandpackExample
+                code={collapsibleWithMixedIconsExample}
+                name="Collapsible mixed icons example"
+                previewHeight={500}
+              />
+            }
+          />
+          <MainSection.Card
+            cardSize="lg"
+            title="Collapsible header"
+            sandpackExample={
+              <SandpackExample
+                code={collapsibleHeaderExample}
+                name="Collapsible header example"
+                previewHeight={500}
+              />
```

Comment:
Can we add an example for mobile? Is any of this supported on mobile? If not, we should describe what;s not supported on the Mobile section.

Reply:
I added the note in the props 😬. Can we make a follow up PR for this?

Reply:
descriptions are added